replit/river
# Various API cleanup, testing

- test proxy case
- deprecate stdio transport
- remove `serverId` arg from `createClient` to reduce confusion as we can derive the ID from the transport passed in
- rename bootSequence -> handshakeSequence
- don't need callback for some handshake related functions
- check protocol version in handshake
